tensorflow/lite/kernels/slice.cc

```diff
@@ -254,6 +254,12 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
     case kTfLiteString:
       TF_LITE_SLICE(string);
       break;
+    case kTfLiteFloat16:
+      TF_LITE_SLICE(Eigen::half);
```

Comment:
Please add include for Eigen/core

Reply:
Addressed the requested change in [#55aee76](https://github.com/MCW-Dev/tensorflow/pull/33/commits/55aee761ece414a6958216f60479cd0358f1b3f3)